### DISCUSSION TITLE
Jigsaw puzzle API

### DISCUSSION THREAD

**Discussion by Rickyy09** [ImageMagick/ImageMagick#7866] [General]
Hi, my name is Ricky, I'm creating a mobile game that allows users to build jigsaw puzzles out of photos. However considering I have no coding experience I need a API to help me break the photos down into jigsaw pieces. How can I do this?

**Comment by snibgo** [comment]
An internet search for "jigsaw puzzle API" finds some results.

**Comment by fmw42** [comment]
What is your version of Imagemagick and on what platform/OS?
